Library/Formula/node.rb

```diff
@@ -82,6 +82,8 @@ def install
       ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
 
       cd buildpath/"npm_install" do
+        # Patch node-gyp until github.com/TooTallNate/node-gyp/pull/564 is resolved
+        patch :DATA
```

Comment:
Shit, your right.  Sorry, my test must have misled me.  Fixing for discussions sake.